arch/xtensa/soc/esp32/linker.ld

```diff
@@ -30,6 +30,9 @@ PROVIDE ( esp32_rom_uart_attach = 0x40008fd0 );
 PROVIDE ( esp32_rom_intr_matrix_set = 0x4000681c );
 PROVIDE ( esp32_rom_gpio_matrix_in = 0x40009edc );
 PROVIDE ( esp32_rom_gpio_matrix_out = 0x40009f0c );
+PROVIDE ( esp32_rom_Cache_Flush = 0x40009a14 );
```

Comment:
Nitpick: lower case names, please.

Reply:
That's the name in the Espressif header though, even though we don't use it from there.

Reply:
nothing like mixing up hump notation/CamelCase and underscore notation.
I say leave it to be consistent with the HAL.